### PR TITLE
Fee selection added for channel requests operations

### PR DESCRIPTION
**Pull request by markettes**

### CODE REVIEW

File: src/Helpers/Constants.cs [hunk 1]
```diff
@@ -27,7 +27,7 @@ public class Constants
     public static readonly bool ENABLE_REMOTE_SIGNER;
     public static readonly bool PUSH_NOTIFICATIONS_ONESIGNAL_ENABLED;
     public static readonly bool ENABLE_HW_SUPPORT;
-    public static readonly bool FEE_SELECTION_ENABLED = false; // Incomplete feature, remove this line when it's ready
+    public static readonly bool FEE_SELECTION_ENABLED = true; // Incomplete feature, remove this line when it's ready
```
Comment (RodriFS) [inline src/Helpers/Constants.cs:30]: You can remove this

**Review by RodriFS — APPROVED** [review]

**Review by RodriFS — APPROVED** [review]

File: src/Pages/ChannelRequests.razor [hunk 2]
```diff
@@ -453,9 +455,10 @@
 
     private async Task OnShowNewChannelRequestModal()
     {
+        _recommendedFees = await NBXplorerService.GetMempoolRecommendedFeesAsync();
         _utxoSelectorModalRef.ClearModal();
         FeesSelection = 0;
-        FeeAmount = 0;
+        OnChangeFeeSelection(FeesSelection);
```
Comment (Jossec101) [inline src/Pages/ChannelRequests.razor:468]:
```suggestion
       await OnChangeFeeSelection(FeesSelection);
```

File: src/Pages/ChannelRequests.razor [hunk 3]
```diff
@@ -273,6 +270,13 @@
                 }
             </DisplayTemplate>
         </DataGridColumn>
+        <DataGridColumn TItem="ChannelOperationRequest" Field="@nameof(ChannelOperationRequest.FeeRate)" Caption="Private" Displayable="true">
+            <DisplayTemplate>
+                @{
+                    @($"{context.FeeRate} sats/vbyte")
```
Comment (Jossec101) [inline src/Pages/ChannelRequests.razor:276]: Is this nullable?

Comment (markettes) [reply src/Pages/ChannelRequests.razor]: Yes!

**Review by RodriFS — APPROVED** [review]

**Pull request merged**